modules/shadertools/src/lib/filters/prop-types.ts

```diff
@@ -26,14 +26,16 @@ const TYPE_DEFINITIONS = {
     }
   },
   array: {
-    validate(value: unknown, propType: PropDef) {
+    validate(value: unknown, propType: PropType) {
       return Array.isArray(value) || ArrayBuffer.isView(value);
     }
   }
 };
 
-export function parsePropTypes(propDefs: Record<string, PropDef>): Record<string, PropType> {
-  const propTypes: Record<string, PropType> = {};
+
+
+export function parsePropTypes(propDefs: Record<string, PropType>): Record<string, PropDef> {
```

Comment:
I find the types and parameter naming confusing. It's not clear why you chose to swap the names of the types but kept the parameter names.

Reply:
Good catch. Overhauled in same PR #1679